### PR TITLE
Issue SB SB-5512 feat: Documenting Upgrades (Angular V6)

### PR DESCRIPTION
**Pull request by anoophm**
- [ ] Have you followed [Contribution Guidelines](http://www.sunbird.org/contributions/contribution_guidelines/)

-----

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @anoophm,<br/><br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/519. <br/><br/> **Modified pages are -** <br />- http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/519/developer-docs/upgrade/upgrade_angular_node <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @anoophm,<br/><br/> _This is a auto-generated response._ <br/> Click the URL for review pull request - http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/519. <br/><br/> **Modified pages are -** <br />- http://sunbird-docs-qa.s3-website.ap-south-1.amazonaws.com/pr/519/developer-docs/upgrade/upgrade_angular_node <br />  <br/> **Modified sections are -** <br />

File: pages/developer-docs/upgrade/upgrade_angular_node.md [hunk 1]
```diff
@@ -1,39 +1,36 @@
...
-
- <List down the Imports, usage of Creation methods, types, and the operators that will change after upgrade> 
- 
-## Configuration Parameters (if any)
-
-<this section should provide configuration parameter details, if any. If there are no specific details, this section can be deleted.>
-
+## Steps to update Angular v5 to v6:
 
+*  take latest sunbird-portal release v1.9 or above
```
Comment (nawabahmadreshi) [inline pages/developer-docs/upgrade/upgrade_angular_node.md]: Please mention from where ?

**Comment by Basreena** [comment]
Merging without review. @nawabahmadreshi, please edit the file and send a fresh PR

**Pull request merged**